tests/Gaufrette/Functional/Adapter/AzureMultiContainerBlobStorageTest.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Gaufrette\Functional\Adapter;
+
+/**
+ * Class AzureMultiContainerBlobStorageTest
+ * @group AzureBlobStorage
+ * @group AzureMultiContainerBlobStorage
+ */
+class AzureMultiContainerBlobStorageTest extends FunctionalTestCase
```

Comment:
$ vendor/bin/phpunit --group AzureBlobStorage --testdox
    PHPUnit 3.7.38 by Sebastian Bergmann.
    
    Configuration read from /home/dado/Development/Gaufrette/phpunit.xml.dist
    
    Gaufrette\Functional\Adapter\AzureBlobStorage
    [x] Should write and read
    [x] Should update file content
    [x] Should check if file exists
    [x] Should get mtime
    [x] Should fail when try mtime for key which does not exist
    [x] Should rename file
    [x] Should delete file
    [x] Should fetch keys
    [x] Should work with hidden files
    [x] Should keep file object in register
    [x] Should wrtie to same file
    
    Gaufrette\Functional\Adapter\AzureMultiContainerBlobStorage
    [x] Should write and read
    [x] Should update file content
    [x] Should check if file exists
    [x] Should get mtime
    [x] Should fail when try mtime for key which does not exist
    [x] Should rename file
    [x] Should delete file
    [x] Should fetch keys
    [x] Should work with hidden files
    [x] Should keep file object in register
    [x] Should wrtie to same file